### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by Winminoo4444**
Please review my code @Yuchen513

### CODE REVIEW

File: prac_06/car.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        Drive given distance if car has enough fuel
+        or drive until fuel runs out return the distance actually driven.
+        """
+        if distance > self.fuel:
+            distance = self.fuel
+            self.fuel = 0
+        else:
+            self.fuel -= distance
+        self._odometer += distance
+        return distance
```
Comment (Yuchen513) [inline prac_06/car.py:33]: This task was done very well, I didn't see any problem.

File: prac_06/guitar_test.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from prac_06.guitar import Guitar
+
+guitar = Guitar("Gibson L-5 CES", 1922, 16035.40)
+another_guitar = Guitar("Another Guitar", 2013, 1000)
```
Comment (Yuchen513) [inline prac_06/guitar_test.py:4]: I think using guitar1 and  guitar2 is better

**Comment by Winminoo4444** [comment]
Thank you for your review. Appreciate for the details and giving me opportunity to learn a lot from your review.

**Comment by Winminoo4444** [comment]
Thank you for your review. Wish you luck for your lessons.

**Pull request merged**